### PR TITLE
Origin/feature/profile/ecom 40 terms and conditions

### PR DESCRIPTION
**Pull request by haidyhesham95**
Uploading WhatsApp Video 2024-12-11 at 02.05.22_195b19c0.mp4…

### CODE REVIEW

File: lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart [hunk 1]
```diff
@@ -25,46 +26,54 @@ class CustomBestSellerListState extends State<CustomBestSellerList> {
...
+                            SizedBox(
+                              width: 170.w,
+                              height: 180.h,
+                              child: CachedNetworkWidget(
+                                imageUrl:
+                                    state.bestSeller[index].imgCover ?? '',
+                                fit: BoxFit.cover,
+                              ),
+                            ),
+                            SizedBox(
```
Comment (YasmeenGad) [inline lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart:56]: convert it to vertical spacing built widget

File: lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart [hunk 2]
```diff
@@ -25,46 +26,54 @@ class CustomBestSellerListState extends State<CustomBestSellerList> {
...
+                              width: 170.w,
+                              child: Text(
+                                state.bestSeller[index].title ?? '',
+                                style: MyFonts.styleRegular400_14.copyWith(
+                                    color: MyColors.blackBase,
+                                    overflow: TextOverflow.visible),
+                                maxLines: 2,
+                              ),
+                            ),
+                            SizedBox(
```
Comment (YasmeenGad) [inline lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart:69]: and this

File: lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart [hunk 3]
```diff
@@ -25,46 +26,54 @@ class CustomBestSellerListState extends State<CustomBestSellerList> {
...
+                              height: 3.h,
+                            ),
+                            Text('${state.bestSeller[index].price}',
+                                style: MyFonts.styleMedium500_14
+                                    .copyWith(color: MyColors.blackBase)),
+                          ],
                         ),
```
Comment (YasmeenGad) [inline lib/features/home_screen/presentation/widgets/custom_best_seller_list.dart:76]: add sized box remove it

File: lib/features/home_screen/presentation/widgets/custom_categories_list.dart [hunk 4]
```diff
@@ -60,7 +63,7 @@ class _CustomCategoriesListState extends State<CustomCategoriesList> {
                     Text('${state.categories?[index]!.name}', style: MyFonts.styleRegular400_14.copyWith(
```
Comment (YasmeenGad) [inline lib/features/home_screen/presentation/widgets/custom_categories_list.dart:63]: remove single quotation

**Review by YasmeenGad — APPROVED** [review]

**Review by Marco-Nagy — APPROVED** [review]

**Pull request merged**